.env.sample

```diff
@@ -0,0 +1,2 @@
+env=local
```

Comment:
I added it for consistency with the plugin and because having an .env file without a env variable inside of it seemed odd. That being said, I am absolutely fine with removing it since it is not used directly but more there for extensibility and consistency.